sharness/test_cli_scores.t

```diff
@@ -0,0 +1,86 @@
+#!/bin/sh
+
+# Disable these lint rules globally:
+#   2034 = unused variable (used by sharness)
+#   2016 = parameter expansion in single quotes
+#   1004 = backslash-newline in single quotes
+# shellcheck disable=SC2034,SC2016,SC1004
+:
+
+test_description='tests for scripts/build_static_site.sh'
+
+export GIT_CONFIG_NOSYSTEM=1
+export GIT_ATTR_NOSYSTEM=1
+
+# shellcheck disable=SC1091
+. ./sharness.sh
+
+test_expect_success "environment and Node linking setup" '
+    toplevel="$(git -C "$(dirname "$0")" rev-parse --show-toplevel)" &&
+    snapshot_directory="${toplevel}/sharness/__snapshots__/" &&
+    SOURCECRED_DIRECTORY="${snapshot_directory}/example-github-load" &&
+    snapshot_file="${snapshot_directory}/example-github-scores.json" &&
+    if [ -z "${SOURCECRED_BIN}" ]; then
+        printf >&2 "warn: missing environment variable SOURCECRED_BIN\n" &&
+        printf >&2 "warn: using repository bin directory as fallback\n" &&
+        export SOURCECRED_BIN="${toplevel}/bin"
+    fi &&
+    export NODE_PATH="${toplevel}/node_modules${NODE_PATH:+:${NODE_PATH}}" &&
+    test_set_prereq SETUP
+'
+
+run() (
+  set -eu
+  rm -f out err
+  node "${SOURCECRED_BIN}"/sourcecred.js "$@" >out 2>err
```

Comment:
@wchargin would you be willing to take a look at this file and tell me what you think?

Reply:
Done.